person.rb

```diff
@@ -0,0 +1,27 @@
+require './person'
+
+class Person
```

Comment:
- [ OPTIONAL ] Please from the project requirement, you should make sure that the `Person` class inherits from Nameable :pray: